src/app/ubs/ubs-admin/components/ubs-admin-table/table-cell-readonly/table-cell-readonly.component.scss

```diff
@@ -13,6 +15,18 @@ div {
   padding: 0 20px;
 }
 
+.un-paid {
+  background-color: var(--primary-white) !important;
+}
+
+.paid {
+  background-color: #ff0 !important;
```

Comment:
add color variable here

Reply:
I thought that creating a constant for a color that is not included in our design is pointless)).
Added changes and at the same time made constants for color numbers in the .ts file